utills/show_user_input.py

```diff
@@ -0,0 +1,25 @@
+import streamlit as st
+from openai import OpenAI, load_qa_chain, get_openai_callback
+
+
+# Define a function to display user input and generate responses based on it
+def show_user_input(knowledge_base):
```

Comment:
We need to only have methods that can be imported and used by the dev who is going to use streamlit to build a UI

Reply:
Assume that you are receiving a pdf and a question